app/lib/frontend/handlers/publisher.dart

```diff
@@ -31,8 +33,17 @@ Future<shelf.Response> getPublisherApiHandler(
 /// Handles requests for PUT /api/publisher/<publisherId>
 Future<shelf.Response> updatePublisherApiHandler(
     shelf.Request request, String publisherId) async {
-  // TODO: implement
-  return notFoundHandler(request);
+  final body = await request.readAsString();
+  final data = json.decode(body) as Map<String, dynamic>;
+  final rq = UpdatePublisherRequest.fromJson(data);
+  if (rq.contact != null) {
+    // TODO: implement
+    throw UnimplementedError('Contact change is not implemented yet.');
+  }
+  if (rq.description != null) {
+    await publisherBackend.updatePublisherData(publisherId, rq.description);
```

Comment:
Let's enforce a maximum length of the description... any limit will do... maybe 256kb is sufficient just as a sanity limit.
If the value is higher than 256kb we should return 4xx.

Reply:
I think the backend internal logic should enforce that (which it does and throws ArgumentError). We should have a centralized handler for such exceptions in the handler wrappers though...